### PR TITLE
contact page and api

### PR DESCRIPTION
**Pull request by 8rianHo**
description:
- installed formik and yup for the contact form / wider app

- added the contact page design

-added /api/contact endpoint with middleware to attach id (familyName for now) to the body if the user is logged in with a token

- updated mock token to the more recent one in linear

- updated session storage to use access_token key

### CODE REVIEW

File: pages/contact.js [hunk 1]
```diff
@@ -1,9 +1,194 @@
...
+            <div className="w-full lg:w-1/2 px-4">
+              <div className="px-4 py-8 md:p-10 bg-coolGray-50 rounded-md">
+                <Formik
+                  enableReinitialize
+                  initialValues={{
+                    email: user?.email || "",
+                    message: "",
+                  }}
+                  validationSchema={Yup.object().shape({
+                    email: Yup.string()
```
Comment (lujasmine) [inline pages/contact.js:73]: We could put this in a separate file? Especially for the email one, as we can reuse it in the other forms?

File: pages/api/contact.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import jwt_decode from "jwt-decode";
+
+const checkLoggedInContactMiddleware = (handler) => (req, res) => {
+  const { authorization } = req.headers;
+  if (!authorization) return handler(req, res);
```
Comment (lujasmine) [inline pages/api/contact.js:5]: With this current code, does it return 200 here? I imagine the handler will be updated to call the backend route once it's ready and the behaviour will change? 

Just trying to understand the purpose of the handler here

File: pages/contact.js (hunk 1 above)
Comment (8rianHo) [reply pages/contact.js:73]: good point - I'll create a validation folder of some sort

File: pages/api/contact.js (hunk 2 above)
Comment (8rianHo) [reply pages/api/contact.js:5]: it's the end of the middleware so `return handler(req,res)` is the same as `next()` in express

this is an early return where if there's no authorisation token then we don't need to attach the id to the body, however looking at https://github.com/8rianHo/vietnamese-directory-server/pull/21 and using the email to append the id, this middleware is now redundant right?

Comment (lujasmine) [reply pages/api/contact.js:5]: Ahh cool, makes sense 👌 

Yeah I think we don't need it here now, if we're going for using the email to append the id!

**Review by lujasmine — APPROVED** [review]

**Pull request merged**